src/util.cpp

```diff
@@ -1464,7 +1465,8 @@ std::string RoundToString(double d, int place)
 
 double RoundFromString(const std::string& s, int place)
 {
-    return Round(atof(s.c_str()), place);
+    double num = boost::lexical_cast<double>(s);
```

Comment:
Could you add a comment here that we cannot use atof due to locale issues? So we don't switch back and bump into the same issue again later.